.github/workflows/build.yaml

```diff
@@ -9,11 +9,39 @@ on:
       - master
 
 jobs:
-  hello:
-    name: Hello
+  static-analysis:
+    name: Static Analysis
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+
     steps:
-      - name: Echo a greeting
-        run: echo 'Hello, GitHub Actions!'
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: none
+          php-version: ${{ matrix.php-version }}
+
+      - name: Determine composer cache directory
+        id: determine-composer-cache-directory
+        run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+      - name: Cache dependencies installed with composer
+        uses: actions/cache@v2
```

Comment:
With `actions/cache` we can cache arbitrary directories or files between jobs.

💁‍♂️ For reference, see:

- https://github.com/actions/cache